tiledb/sm/filesystem/s3/STSProfileWithWebIdentityCredentialsProvider.h

```diff
@@ -132,11 +132,15 @@ class /* AWS_IDENTITY_MANAGEMENT_API */
       const Aws::String& externalID,
       Aws::STS::STSClient* client);
 
```

Comment:
We could document this function and the one above, not a blocker as this is high priority.